share/picongpu/examples/TransitionRadiation/etc/picongpu/1.cfg

```diff
@@ -52,8 +52,10 @@ TBG_pngYX="--e_png.period 10 \
            --e_png.slicePoint 0.5 \
            --e_png.folder pngElectronsYX"
 
-# transition radiation plugi
-TBG_e_transRad="--e_transRad.period 10"
+# transition radiation plugin
+TBG_e_transRad="--e_transRad.period 10 \
+                --e_transRad.file transRad \
+                --e_transRad.foilPosition 0.0"
```

Comment:
is a foil at 0 useful?

Reply:
Doesn't that mean `0==disabled`?

Reply:
No, a foil at 0 is physically useless, so I have this is the "the feature is disabled"-value. The point of this virtual foil propagation is, that one can naively "simulate" the vacuum-propagation of an accelerated electron bunch from an LWFA after the plasma stage, without having to run the simulation millimeters to centimeters that would mimic the experimental setup. The evolution of the bunch during the plasma stage is already handled much better by the PIC loop itself, if you want to know how the CTR spectra look before the current bunch position, you just call the plugin earlier. 
I was struggling a bit with what the clearest way is to have this optional parameter. Maybe I should add a new parameter called `--e_transRad.foilEnabled 0` to separate the activation and the parameter of this feature.